service/src/test/java/bio/terra/workspace/service/resource/controlled/ControlledResourceServiceTest.java

```diff
@@ -342,15 +345,7 @@ void createAiNotebookInstanceUndo() throws Exception {
             .instanceId(instanceId)
             .location(DEFAULT_NOTEBOOK_LOCATION);
     ControlledAiNotebookInstanceResource resource =
-        ControlledResourceFixtures.makeDefaultAiNotebookInstance()
-            .workspaceId(workspace.getWorkspaceId())
-            .name(instanceId)
-            .accessScope(AccessScopeType.ACCESS_SCOPE_PRIVATE)
-            .managedBy(ManagedByType.MANAGED_BY_USER)
-            .assignedUser(user.getEmail())
-            .instanceId(instanceId)
-            .location(DEFAULT_NOTEBOOK_LOCATION)
-            .build();
+        makeNotebookTestResource(workspace.getWorkspaceId(), instanceId, instanceId);
```

Comment:
nit: probably better to have a name distinct from the id so we don't accidentally miss a crossed wire or something.